nightingale_interface/screens/wait_item_get_screen.py

```diff
@@ -0,0 +1,59 @@
+from kivy.uix.screenmanager import Screen
+from kivy.uix.image import Image
+from kivy.uix.videoplayer import VideoPlayer
+
+from kivymd.uix.label import MDLabel
+from kivymd.uix.button import MDRectangleFlatButton
+from kivy.uix.button import Button
+
+from kivy.uix.screenmanager import SlideTransition, NoTransition
+from screens.screen_config import ScreenConfig as cfg
+
+
+class WaitItemGetScreen:
+    def estop(self, button_data):
+        button_data.parent.manager.transition = NoTransition()
+        cfg.LAST_SCREEN = "waititemgetscreen"
+        button_data.parent.manager.current = "estoppedscreen"
+
+    def retract_arm(self, button_data):
+        # publishes message to stop to retract arm
+        button_data.parent.manager.transition = NoTransition()
+        cfg.LAST_SCREEN = "waititemgetscreen"
+        button_data.parent.manager.current = "retractarmscreen"
+
+    def wait_item_get_build(self):
+        screen = Screen(name="waititemgetscreen")
+
+        # estop button
+        screen.add_widget(
+            Button(
+                background_normal="images/stop.png",
+                size_hint_x=cfg.ESTOP_XHINT,
+                size_hint_y=cfg.ESTOP_YHINT,
+                pos_hint={"center_x": cfg.ESTOP_XPOS, "center_y": cfg.ESTOP_YPOS},
+                on_release=self.estop,
+            )
+        )
+
+        # retract arm
+        screen.add_widget(
+            MDRectangleFlatButton(
+                text="Finish",
+                font_style="H4",
+                pos_hint={"center_x": 0.85, "center_y": 0.15},
+                size_hint=(0.4, 0.12),
+                on_release=self.retract_arm,
+            )
+        )
+
```

Comment:
add button to say item retrieval is done. Will pressing buttons put the patient in harm's way if arm is expected to be moving? even after some delay to allow patient to move away?

Reply:
that is a good point, i feel like all arm control should probably be in the admin screen

Reply:
I talked a bit to Dr DeHart today about it and he suggested that we need some form of user input during the item retrieval process to make it safe and allow the user to be aware of the robot movement and control the process. He suggested either voice recognition, buttons, or gestures and I was thinking that button presses are the easiest to implement for a base application. If we want to incorporate something else the procedure of handing off will be the same(same screens) and we can just swap out the buttons for gesture detection or voice recognition. I think we should keep this here to give the user control during the item retrieval.